images/scripts/fedora.setup

```diff
@@ -91,7 +91,7 @@ libreport-plugin-bugzilla \
 libreport-fedora
 "
 
-if [ "$IMAGE" = "fedora-34" ]; then
```

Comment:
Can you completely invert the logic here?  ie: not only change the check to "is this fedora-33?", but also change the list above to contain `ntpsec` and change the code here to replace `ntpsec` with `ntp`, with appropriately updated comments?

That way when fedora-33 drops out, we can just delete this section.

Reply:
Yes, very good point.

Reply:
Done.